### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,6 +23,22 @@
...
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.5.2</version>
+        </dependency>
```
Comment (vicire) [inline pom.xml:35]: You need one of these two dependecies

File: src/test/java/CustomExecutorTest.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    @Test
+    public void getSum_Ok() {
+        Integer expected = NumbersProducer.getIntegerList().stream()
+                .reduce(Integer::sum).get();
+        Integer actual = customExecutor.getSum();
+        assertEquals(expected, actual, "Expected: " + expected + "Actual: " + actual);
+    }
+}
```
Comment (vicire) [inline src/test/java/CustomExecutorTest.java]: add empty line

File: src/test/java/CustomForkJoinTest.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    @Test
+    public void compute_Ok() {
+        Integer expected = NumbersProducer.getIntegerList().stream()
+                .reduce(Integer::sum).get();
+        Integer actual = customForkJoin.compute();
+        assertEquals(expected, actual, "Expected: " + expected + "Actual: " + actual);
+    }
+}
```
Comment (vicire) [inline src/test/java/CustomForkJoinTest.java]: same here

File: src/test/java/CustomForkJoinTest.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    @BeforeAll
+    public static void setUp() {
+        list = new NumbersProducer().getIntegerList();
+        customForkJoin = new CustomForkJoin(list);
+    }
+
+    @Test
+    public void compute_Ok() {
+        Integer expected = NumbersProducer.getIntegerList().stream()
+                .reduce(Integer::sum).get();
```
Comment (vicire) [inline src/test/java/CustomForkJoinTest.java]: You can put this logic in setUp

File: src/test/java/CustomExecutorTest.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    @BeforeAll
+    public static void setUp() {
+        list = new NumbersProducer().getIntegerList();
+        customExecutor = new CustomExecutor(list);
+    }
+
+    @Test
+    public void getSum_Ok() {
+        Integer expected = NumbersProducer.getIntegerList().stream()
+                .reduce(Integer::sum).get();
```
Comment (vicire) [inline src/test/java/CustomExecutorTest.java]: Same here

**Review by vicire — CHANGES_REQUESTED** [review]

**Review by vicire — APPROVED** [review]

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**